generate-wiki-summary.php

```diff
@@ -74,7 +74,8 @@
 
 		// Summary
 		if($page['type'] == 'new') {
-			if(preg_match('/^user:/i', $title) && !preg_match('/\.css$/i', $title))
+			//subpages of user-pages are excluded (e.g. user css or personal test pages)
```

Comment:
Probably want `// Subpages` instead of `//subpages`, for consistency with other comments